product_docs/docs/epas/16/epas_rel_notes/epas16_rel_notes.mdx

```diff
@@ -9,7 +9,7 @@ EDB Postgres Advanced Server 16.1 includes the following enhancements and bug fi
 
 |      Type      |                                                                                                                                                                                 Description                                                                                                                                                                                 | Category |
 | -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
-| Upstream merge | Merged with community PostgreSQL 16 beta. See the [PostgreSQL 16 Release Notes](https://www.postgresql.org/docs/16/release-16.html) for more information.                                                                                                                                                                                                                   |          |
+| Upstream merge | Merged with community PostgreSQL 16.1. See the [PostgreSQL 16 Release Notes](https://www.postgresql.org/docs/16/release-16.html) for more information.                                                                                                                                                                                                                   |          |
```

Comment:
Is the link for PG right? or it should be https://www.postgresql.org/about/news/postgresql-161-155-1410-1313-1217-and-1122-released-2749/ ?

Reply:
```suggestion
| Upstream merge | Merged with community PostgreSQL 16.1. See the [PostgreSQL 16 Release Notes](https://www.postgresql.org/docs/16/release-16-1.html) for more information.                                                                                                                                                                                                                   |          |
```